master/templates/conf-template.conf

```diff
@@ -1,10 +1,16 @@
-{% if config.pop('nested', True) %}
-{{ config.name }}:
-  {% for arg, val in config.params.items() %}
-  - {{ arg }}: {{ val }}
-  {% endfor %}
-{% else %}
-{% for arg, val in config.items() %}
-{{ arg }}: {{ val }}
-{% endfor %}
-{% endif %}
\ No newline at end of file
+{%- macro render_yaml(config_dict, offset=2) -%}
```

Comment:
Is this macro defined within our repos somewhere? I looked for it in the docs but couldn't seem to find it: https://docs.saltstack.com/en/develop/ref/renderers/all/salt.renderers.jinja.html